packages/flutter/lib/src/material/tabs.dart

```diff
@@ -395,9 +395,11 @@ class _IndicatorPainter extends CustomPainter {
 
     if (indicatorSize == TabBarIndicatorSize.label) {
       final double tabWidth = tabKeys[tabIndex].currentContext!.size!.width;
-      final double delta = ((tabRight - tabLeft) - tabWidth) / 2.0;
-      tabLeft += delta;
-      tabRight -= delta;
+      final EdgeInsetsGeometry labelPadding = tabKeys[tabIndex].currentContext!.findAncestorRenderObjectOfType<RenderPadding>()!.padding;
+      final EdgeInsets insets = labelPadding.resolve(_currentTextDirection);
+      final double delta = ((tabRight - tabLeft) - (tabWidth + insets.horizontal)) / 2.0;
+      tabLeft += delta + insets.left;
```

Comment:
tabLeft += (delta + insets.left)

Reply:
it seems that () is not necessary, 'Unnecessary parenthesis can be removed(unnecessary_parenthesis).'